res/xml/qwerty.xml

```diff
@@ -32,13 +32,13 @@
     <key key0="b" key2="\?" key3="/"/>
     <key key0="n" key1="accent_tilde" key2=":" key3=";"/>
     <key key0="m" key2="&quot;" key3="'"/>
-    <key width="1.5" key0="backspace" key2="delete"/>
+    <key width="1.5" key0="backspace" key1="delete"/>
   </row>
   <row height="0.95">
-    <key width="1.8" key0="ctrl" key3="switch_numeric"/>
+    <key width="1.8" key0="ctrl" key2="switch_numeric"/>
```

Comment:
After a bit of testing, I'd say `key4` is easier to hit for this key.

Reply:
I guess it works too. But I liked the idea of swiping towards the center of the keyboard to change it's layout to numbers, almost like bringing the numbers, if there was an animation for the transition. I'll have to test on position four.

Reply:
Nice justification :)